packages/xl-ai/src/AIExtension.ts

```diff
@@ -297,6 +298,24 @@ export class AIExtension extends BlockNoteExtension {
     }
   }
 
+  /**
+   * Abort the current LLM request.
+   *
+   * Only valid when executing an LLM request (status is "thinking" or "ai-writing")
+   */
+  public async abort() {
+    const state = this.store.getState().aiMenuState;
+    if (state === "closed") {
+      throw new Error("abort() is only valid during LLM execution");
+    }
+    if (state.status !== "thinking" && state.status !== "ai-writing") {
+      throw new Error("abort() is only valid during LLM execution");
+    }
```

Comment:
I think this should be an early return, user-facing functionality should not throw errors when it is trivial to know.

This would be similar to having to check if you can abort something before you abort it. It should just no-op